scanpy/datasets/_datasets.py

```diff
@@ -377,6 +389,7 @@ def visium_sge(
         'Targeted_Visium_Human_ColorectalCancer_GeneSignature',
         'Parent_Visium_Human_ColorectalCancer',
     ] = 'V1_Breast_Cancer_Block_A_Section_1',
+    download_tissue_image: bool = False,
```

Comment:
Instead of `tissue_image`, could this be something like `tiff`, `hires_tiff`, or `raw_tiff`?

I don't like `tissue_image`. The jpegs we load by default could also be called `tissue_image`s, so this seems a little unclear to me.

Reply:
sure, I can call this flag `download_hires_tiff`. This is only for downloading images from 10x, and they only provide hires tiff images.
@giovp made the point earlier that in general, the hires_tiff image could also be a jpg, so I would still call the entry in `adata.uns` something like `tissue_image_path` or `hires_image_path`.

Reply:
agree, argument of download could have `tiff` but argument in `uns` should not. I would also drop hires cause there is already a hires in `adata.uns["spatial"]`. Maybe `full_tissue_image`  or something like that? don't know sorry

Reply:
What uses these images in `squidpy`? In particular what uses these images that wouldn't work with the `jpeg`s?

Reply:
in `squidpy` this image is used by the image container. Don't understand the `jpeg` question sorry

Reply:
Ha, for `jpeg` I meant the `tissue_hires_image.png` and `tissue_lowres_image.png`.

Reply:
I think the implication is that this is the `raw`, `full`. or `source` image that you want to do processing on.

I would agree that `hires` would also be a good descriptor, but that's already used by the `png`.

You would be in a better position to comment, but I like `source_image_path` and `full_image_path`. Both imply that they are the complete data, which is what you'll want to do processing on.

Reply:
ah ok, so 
> what uses these images that wouldn't work with the jpegs [pngs]?

basically everything that works on imag in squidpy, in particular segmentation, feature extraction and visualization with napari

> You would be in a better position to comment, but I like source_image_path and full_image_path. Both imply that they are the complete data, which is what you'll want to do processing on.

Also agree, actually `source_image_path` is nice as that image is indeed the source image for spaceranger to do the processing